x-pack/plugins/infra/public/components/logging/log_analysis_results/analyze_in_ml_button.tsx

```diff
@@ -95,10 +95,7 @@ const getPartitionSpecificSingleMetricViewerLink = (
     },
   });
 
-  const hash = `/timeseriesexplorer?${stringify(urlUtils.encodeQuery({ _g, _a }), {
-    sort: false,
-    encode: false,
-  })}`;
+  const hash = `/timeseriesexplorer?${stringify(urlUtils.encodeQuery({ _g, _a }))}`;
```

Comment:
Same as above, this changes the behaviour of these links. I think we can fix this with:

```suggestion
  const hash = `/timeseriesexplorer?${makeUrlFromQuery({ _g, _a })}`;
```